__tests__/components/editor/InputLookupQA.test.js

```diff
@@ -13,6 +13,7 @@ const plProps = {
       "type": "lookup",
       "resourceTemplates": [],
       "valueConstraint": {
+        "repeatable": "true",
```

Comment:
Not a big deal, but am curious why this was added.

Reply:
See line 58 comment on `InputListLOC.jsx` below